src/base/utils/foreignapps.cpp

```diff
@@ -108,18 +113,17 @@ namespace
         if (res == ERROR_SUCCESS)
         {
             ++cMaxSubKeyLen; // For null character
-            LPWSTR lpName = new WCHAR[cMaxSubKeyLen];
+            std::wstring lpName(cMaxSubKeyLen + 5, 0);
             DWORD cName;
```

Comment:
Why `+ 5`?

Reply:
> Why `+ 5`?

Oops :( I used to make a buffer "a bit larger" that way... Well, it seems to be confused. 
And I checked cppref, (since C++11) the interval of `data()` is a closed.  Maybe `std::wstring lpName(cMaxSubKeyLen, 0)` is enough.